pkg/internal/ebpf/common/spanner.go

```diff
@@ -95,3 +101,45 @@ func extractTraceparent(traceparent [55]byte) string {
 	}
 	return string(traceparent[:])
 }
+
+func getSQLOperationAndTable(queryString string) (string, string) {
+	fields := strings.Fields(queryString)
+	if len(fields) == 0 {
+		return "", ""
+	}
+	operation := strings.ToUpper(fields[0])
+	if len(fields) < 2 {
+		return operation, ""
+	}
+	table := ""
+	if operation == "UPDATE" {
```

Comment:
I don't think this would work always, there can be operators for UPDATE, e.g. UPDATE TOP ... PERCENT...

Reply:
This has been refactored to use new sqlprune package.